### ISSUE TITLE
Discussion: Wrapping raw xhtml text with html macro renders better than "--clean-adoc"

### ISSUE THREAD

**Issue by dcwangmit01** [philproctor/confluence-cli#2]
@philproctor,

I was debugging an issue where XHTML uploaded by confluence-cli required the "--clean-adoc" parameter which stripped all embedded SVGs and PNGs.  Without "--clean-adoc", the file upload would error out.  With "--clean-adoc", all of the embedded base64 images are stripped.

I found this note here regarding the HTML macro which allows for the XHTML (created via asciidoctor) to be presented untouched.

https://community.atlassian.com/t5/Answers-Developer-Questions/Confluence-create-page-with-custom-image/qaq-p/471978#M9634

Thus, if we wrap the XHTML contents with the following header and footer, the entire XHTML may be uploaded and rendered in its entirety.

```
<ac:structured-macro ac:name = "html"><ac:plain-text-body><![CDATA[

    PUT_XHTML_CONTENTS_HERE

]]></ac:plain-text-body></ac:structured-macro>
```

I've tested this BTW.  If PR'able, I'll make one.

**Comment by philproctor** [comment]
I would accept this PR, I don't do much with this project (as evidenced by the slim commit history) but do passively maintain it a bit. If this is shown to be working, it would be much better than the `--clean-adoc` which in a lot of ways mutilates the upload contents.

If I recall correctly, I looked into doing the macro approach when I first made this project but it was not an option at the time, but I could be wrong.
